gateway/repository_gateway.py

```diff
@@ -39,3 +39,7 @@ def has_object(self, id: str) -> bool:
     @abstractmethod
     def get_object_files(self, id: str, include_staged: bool = False) -> list[ObjectFile]:
         pass
+
+    @abstractmethod
+    def log(self, id: str, reversed: bool = True) -> list[VersionInfo]:
+        pass
```

Comment:
I know the underlying command is `log`, but I'd like to see this method be named something more specific, like `get_versions`.

Reply:
It doesn't look like `reversed` is actually used anywhere?

Reply:
I dislike the re-using `versions`, because it's not versions. It's a log of versions.

Reply:
OK, I see your point, that's fine. I just wish the method said something about how it has something to do with version data. Maybe that's implied by logging an object, but feels vague at this point.